ls1intum/Artemis
# Upvote / Downvote questions

### Checklist

### Motivation and Context


Currently students have no possibility to show that they are having the same question or to rate questions in general.

### Description


Students can upvote / downvote questions to express (dis-)agreement.
There are no negative votes. If the vote is at 0 a student cannot downvote the question.
Questions are ordered by their votes.

### Steps for Testing


1. Log in to Artemis
(1a. Enable questions and answers for your test course if not already enabled) 
2. Navigate to an exercise or lecture
3. Create multiple questions
4. Upvote / Downvote questions in any order
5. Check if sorting and representation works correctly

### Screenshots


default:
<img width="39" alt="Bildschirmfoto 2020-05-25 um 18 21 48" src="https://user-images.githubusercontent.com/51987021/82829952-cb47be80-9eb4-11ea-80a2-53abb24b4c82.png">

upvoted:
<img width="50" alt="Bildschirmfoto 2020-05-18 um 18 37 31" src="https://user-images.githubusercontent.com/51987021/82238034-e8214680-9936-11ea-9b4a-f8837778867f.png">
downvoted:
<img width="41" alt="Bildschirmfoto 2020-05-25 um 18 23 31" src="https://user-images.githubusercontent.com/51987021/82829997-de5a8e80-9eb4-11ea-95db-0c1657af1ee4.png">
